lua/nvim-treesitter/locals.lua

```diff
@@ -11,7 +11,7 @@ local M = {
   locals = {}
 }
 
-function M.collect_locals(bufnr)
+function M.collect_locals(bufnr, root)
```

Comment:
Why adding this ? I don't really get it...

Reply:
I added this because I was using this so I could run a query using a node other than the root node of the tree. When we get the definition of a node I was passing in the scope node for the definition and using all references from that. If I don't use this in the definition/usages I will remove it.

Reply:
Okay I get it, LGTM